hnn_core/viz.py

```diff
@@ -273,6 +273,18 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
     elif average:
         dpl = dpl + [average_dipoles(dpl)]
 
+    if type(dpl) == list:
```

Comment:
use `isinstance` for type checks ... and in any case object equality is not done using `==`. You should use `is`.

Reply:
Sorry, I missed the L271. This condition is reduntant and also the else block. I have updated the changes. Please check them